src/components/MainComponent.vue

```diff
@@ -31,7 +31,10 @@
               <BreakdownComponent v-if="item === 'languages'" :metrics="metrics" :breakdownKey="'language'"/>
               <BreakdownComponent v-if="item === 'editors'" :metrics="metrics" :breakdownKey="'editor'"/>
               <CopilotChatViewer v-if="item === 'copilot chat'" :metrics="metrics" />
-              <ApiResponse v-if="item === 'api response'" :metrics="metrics" />
+              <div v-if="isScopeOrganization">
+                <SeatsAnalysisViewer v-if="item === 'seat analysis'" :seats="seats" />
```

Comment:
@DevOps-zhuang I added this condition so it will only display if in the organization scope

Reply:
great！ it is better than my code. where I set the v-if to the <SeatsAnalysisViewer>, and it always display the SeatsAnalysisViewer, thouth there is no content when the scopt is not organize, I will accept your code, thanks)